assets/js/blocks/cart-checkout/mini-cart-contents/editor.scss

```diff
@@ -0,0 +1,18 @@
+.wp-block-woocommerce-mini-cart-contents {
+	max-width: 512px;
```

Comment:
Should we use the same value as the drawer width?

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/ebae6b6da53924998d698991c5042148ad1b7484/assets/js/base/components/drawer/style.scss#L2